src/main/resources/hudson/plugins/plot/PlotAction/index.jelly

```diff
@@ -4,6 +4,7 @@
  * The copyrights to the contents of this file are licensed under the MIT License (http://www.opensource.org/licenses/mit-license.php)
  */
  -->
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:layout title="${it.project.displayName} plots">
     <st:include it="${it.project}" page="sidepanel.jelly" />
```

Comment:
So, here you'd change it it.job which will call `getJob()` but still return the project